content/topics/How-to-ask-for-help-with-your-code/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: how to ask for help with your code
```

Comment:
directory name capitalisation matters. Be consistent with everything else that already exists unless there is a very good reason not to be.

Right now links to this content will break because the automatically generated links expect lower-case folder names.

Also, capitalisation on the title matters. 

title: How to ask for help with your code